.azure-pipelines/win32-conda-CI.yml

```diff
@@ -62,6 +62,12 @@ jobs:
       pytest tests --doctest-modules --junitxml=junit/test-results.xml
     displayName: 'pytest'
 
+  - script: |
+      coverage run --include=keras2onnx/** tests/** --omit=keras2onnx/ktf2onnx/**
```

Comment:
seems that this script cannot work

Reply:
Including only tests/*.py now! --omit should work after --include, so it will only include files not omitted in ktf2onnx/*